build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -146,6 +159,33 @@ public void PublishSharedFramework(string outputRootDirectory, string commitHash
             var content = $@"{commitHash}{Environment.NewLine}{version}{Environment.NewLine}";
             File.WriteAllText(Path.Combine(sharedFrameworkNameAndVersionRoot, ".version"), content);
 
+            // Copy symbols to publish folder
+            List<string> pdbFiles = new List<string>();
+            string symbolsRoot = Path.Combine(_repoRoot, "pkg", "bin", "symbols");
+            string libPdbPath = GetNetCoreAppRuntimeLibSymbolsPath(symbolsRoot, _sharedFrameworkRid, _sharedFrameworkTarget);
+            string nativePdbPath = GetNetCoreAppRuntimeNativeSymbolsPath(symbolsRoot, _sharedFrameworkRid);
+            string toolsPdbPath = GetNetCoreAppToolsSymbolsPath(symbolsRoot);
+            if (Directory.Exists(libPdbPath))
+            {
+                pdbFiles.AddRange(Directory.GetFiles(libPdbPath));
+            }
+            if(Directory.Exists(nativePdbPath))
+            {
+                pdbFiles.AddRange(Directory.GetFiles(nativePdbPath));
+            }
+            if (Directory.Exists(toolsPdbPath))
+            {
+                pdbFiles.AddRange(Directory.GetFiles(toolsPdbPath));
+            }
+            foreach (string pdbFile in pdbFiles)
+            {
+                string destinationPath = Path.Combine(sharedFrameworkNameAndVersionRoot, Path.GetFileName(pdbFile));
+                if (!File.Exists(destinationPath))
```

Comment:
It looks like the native assemblies publish their symbols on nix (like dbg files).  This was causing a collision and I thought it prudent to prefer whatever was already present in the publish folder..